.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "dropbox-sdk-java/src/main/stone"]
-	path = dropbox-sdk-java/src/main/stone
-	url = https://github.com/dropbox/dropbox-api-spec.git
-[submodule "dropbox-sdk-java/stone"]
-	path = dropbox-sdk-java/stone
+[submodule "dropbox/stone"]
+	path = dropbox/stone
 	url = https://github.com/dropbox/stone.git
+[submodule "dropbox/src/main/stone"]
```

Comment:
should we take this opportunity to rename the api spec folder so its not overloaded?

Reply:
Rename it to what?

Honestly, what I think would be better is not having a submodule for this and publishing an `api-spec` artifact in a different module that this can consume as a dependency. That would isolate the stone tooling requirements to that other artifact and make dev on the SDK easier.

Reply:
I did a little looking and this actually has to be named this way since it's a source set, and the StonePlugin looks for a sourceset named `stone`.